app/src/main/java/org/novinger/jason/demoreader/ArticleAdapter.java

```diff
@@ -53,7 +52,7 @@ public ArticleAdapter(ArrayList<Article> articles) {
     public void onBindViewHolder(ViewHolder holder, int position) {
         Article article = mArticles.get(position);
         holder.mHeadline.setText(article.getHeadline());
-        holder.mByline.setText(article.getPrimaryAuthor().toString());
+        holder.mByline.setText(article.getPrimaryAuthor().getName());
         Glide.with(holder.mContext).load(article.getLeadArtURL()).into(holder.mLeadArt);
     }
```

Comment:
interesting https://inthecheesefactory.com/blog/get-to-know-glide-recommended-by-google/en